examples/pylab_examples/fancybox_demo.py

```diff
@@ -152,7 +152,7 @@ def test4(ax):
     ax.add_patch(p_fancy)
 
     ax.text(0.1, 0.8,
-            ' boxstyle="round,pad=0.3"\n mutation\\_aspect=.5',
+            ' boxstyle="round,pad=0.3"\n mutation_aspect=.5',
```

Comment:
I guess we don't have to worry about these getting rendered as math now? Or is that intended behavior now?